python/pyspark/sql/column.py

```diff
@@ -329,6 +329,8 @@ def cast(self, dataType):
         [Row(ages=u'2'), Row(ages=u'5')]
         >>> df.select(df.age.cast(StringType()).alias('ages')).collect()
         [Row(ages=u'2'), Row(ages=u'5')]
+        
+	(Note : astype is alias for cast)
```

Comment:
I think @marmbrus also suggested that cast docs be updated to note the alias the other way -- there may be two more changes to make.